pkgs/top-level/aliases.nix

```diff
@@ -787,6 +787,7 @@ mapAliases ({
   rarian = throw "rarian has been removed as unused"; # Added 2023-07-05
   rccl = throw "'rccl' has been replaced with 'rocmPackages.rccl'"; # Added 2023-10-08
   rdc = throw "'rdc' has been replaced with 'rocmPackages.rdc'"; # Added 2023-10-08
+  redoc-cli = redocly-cli; # Added 2023-12-05
```

Comment:
Thinking about this some more, maybe the alias doesn't really make sense, since the `mainProgram` name is different. Should we just remove this (broken) package?